connectors/byoei.py

```diff
@@ -372,9 +372,7 @@ async def prepare_index(
             return
 
         logger.debug(f"Creating index {index}")
-        await self.client.indices.create(
-            index=index, settings=settings, mappings=mappings
-        )
+        await self.client.indices.create(index=index, mappings=mappings)
```

Comment:
Hm, that is interesting. Looking at the Ruby code, it seems like this code is potentially harmful/can lead to SDH cases.

Some thoughts:
1. This code should never run under normal conditions.
2. This code can run only if somebody/something deleted the content index while syncs are not running.
3. If this code runs, it can create the index with wrong settings UNLESS language code in config file is equal to language code that customer used to create he index via Kibana AND analysers were not modified.

Do you know by chance why this code exists in Ruby? Originally the code to create the index was added to support CLI, but now CLI is not used/supported at all.